src/containers/App.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
```

Comment:
`App.js` shouldn't be in the `containers` folder as it won't be connecting to the redux store